packages/build/src/error/type.js

```diff
@@ -88,7 +88,7 @@ const TYPES = {
 
       return `Bundling of function "${functionName}" failed`
     },
-    group: ({ functionType = 'serverless' }) => `'Bundling of ${functionType} function failed`,
+    group: ({ location: { functionType = 'serverless' } }) => `'Bundling of ${functionType} function failed`,
```

Comment:
This was not working before?

Reply:
No. We weren't capturing this in any snapshots, so we missed it. We are now!

Reply:
TypeScript FTW!

Reply:
💯